### ISSUE TITLE
NTR: MRI cranial nerve enhancement (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9040]
New term request MRI cranial nerve enhancement (gospe_210825153142): Oculomotor nerve; trochlear nerve; abducens nerve

**Comment by pnrobinson** [comment]
@MickeySegal
can you provide more context? Is this enhancement of all nerves or specific nerves? What diseases?

**Comment by MickeySegal** [comment]
Used in " Ophthalmoplegic cranial neuropathy (Ophthalmoplegic migraine)"
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3562350/
"Of patients who had a brain MRI with the administration of gadolinium performed during an acute attack involving the third nerve (n = 52), 75% had contrast enhancement of the third nerve and 76% had nerve thickening. Nerve enhancement of the fourth nerve was seen in one case and in the intraparenchymal portion of the sixth nerve in another case"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000291
- label: Third cranial nerve thickening
- parents: [Abnormal cranial nerve morphology (HP:0001291)](https://hpo.jax.org/app/browse/term/HP:0001291)
- pmid: PMID:22241707
- def: Increase in diameter of the third cranial nerve (oculomotor nerve). This feature can be observed by magnetic resonance imaging and may be accompanied by contrast enhancement.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9040
- synonyms: Oculomotor nerve thickening(NONE)
